packages/network_info_plus/network_info_plus/example/integration_test/network_info_plus_test.dart

```diff
@@ -5,26 +5,59 @@
 // @dart=2.9
 
 import 'dart:io';
+import 'package:flutter/services.dart';
 import 'package:integration_test/integration_test.dart';
 import 'package:flutter_test/flutter_test.dart';
 import 'package:network_info_plus/network_info_plus.dart';
+import 'package:network_info_plus_platform_interface/method_channel_network_info.dart';
 
 void main() {
   IntegrationTestWidgetsFlutterBinding.ensureInitialized();
 
   group('NetworkInfo test driver', () {
+    final log = <MethodCall>[];
     NetworkInfo _networkInfo;
+    MethodChannelNetworkInfo _methodChannelNetworkInfo;
 
     setUpAll(() async {
       _networkInfo = NetworkInfo();
+      _methodChannelNetworkInfo = MethodChannelNetworkInfo();
+
+      _methodChannelNetworkInfo.methodChannel
+          .setMockMethodCallHandler((MethodCall methodCall) async {
+        log.add(methodCall);
+        switch (methodCall.method) {
+          case 'wifiName':
+            return '1337wifi';
+          case 'wifiBSSID':
+            return 'c0:ff:33:c0:d3:55';
+          case 'wifiIPAddress':
+            return '127.0.0.1';
+          case 'wifiIPv6Address':
+            return '2002:7f00:0001:0:0:0:0:0';
+          case 'wifiBroadcast':
+            return '127.0.0.255';
+          case 'wifiGatewayAddress':
+            return '127.0.0.0';
+          case 'wifiSubmask':
+            return '255.255.255.0';
+          case 'requestLocationServiceAuthorization':
+            return 'authorizedAlways';
+          case 'getLocationServiceAuthorization':
+            return 'authorizedAlways';
+          default:
+            return null;
+        }
+      });
+      log.clear();
```

Comment:
This isn't really an integration test, because you are mocking the platform layer. Instead this should be a unit test.

Can you move it to `network_info_plus/test/network_info_test.dart` ?